addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -429,6 +434,7 @@ test("Can reply to history message", async () => {
     await contains(".o_notification", { text: 'Message posted on "RandomName"' });
     await click(".o-mail-DiscussSidebarChannel", { text: "RandomName" });
     await contains(".o-mail-Message-content", { text: "abc" });
+    await contains(".o-mail-Message.o-selfAuthored a.o_mail_redirect", { count: 0 });
```

Comment:
same, would remove or write test to have matching positive selector if you think this is valuable.